oak_attestation_verification/src/lib.rs

```diff
@@ -18,17 +18,24 @@
 
 extern crate alloc;
 
+macro_rules! include_proto {
+    ($package: tt) => {
+        include!(concat!(env!("OUT_DIR"), concat!("/", $package, ".rs")));
+    };
+}
+
 pub mod proto {
     pub mod oak {
-        tonic::include_proto!("oak");
+        include_proto!("oak");
         pub mod attestation {
             pub mod v1 {
-                tonic::include_proto!("oak.attestation.v1");
+                include_proto!("oak.attestation.v1");
```

Comment:
Do we use this marco anywhere else? If so, we could remove the tonic dependency there as well (in a different PR)

Reply:
Yes. tonic::include_proto is used in 12 crates (11 after this). If a crate uses tonic nontrivially then the tonic::include_proto can stay.